app/src/scss/modules/_footer.scss

```diff
@@ -1,211 +1,161 @@
+  place-self: center;
+  padding-top: 30px;
+  a {
+    color: #fff;
+  }
+  @media #{$screen__l} {
+    padding-top: 39px;
+  }
+}
 
-  .cb-grid-container {
+.cb-grid-container {
+  display: none;
+  @media #{$screen__m} {
     display: grid;
     width: 100%;
     height: 100%;
-    //grid-template-columns: 1fr;//1fr
-    //grid-template-rows: 1fr 1fr ; //1fr
     gap: 5px 5px;
-    //justify-self: center;
-    //grid-template-areas:
-      //"CBox-1 CBox-2";  
-      //"CBox-1 CBox-2";
-      @media #{$screen__l} 
-    {
-      grid-template-columns: 1fr 1fr;
-      grid-template-rows: 1fr ; //1fr
-    }
-    li {
-        padding-top:5px;
-    }
-  }
-  
-  .CBox-1 { 
-      //grid-area: CBox-1;
-      place-self: center;
-      a {
-        color: #fff;
-        padding-top: 5px;
-      }
-    }
-    
-  
-  .CBox-2 { 
-      //grid-area: CBox-2; 
-      place-self: center;
-      a {
-        color: #fff;
-        padding: 5px;
-      }
-    }
-  
-  
-  .sb-grid-container {
-    display: grid;
-    width: 100%;
-    height: 100%;
-    //grid-template-columns: 1fr;
-    //grid-template-rows: 1fr;
-    //gap: 0px 0px;
-    grid-template-areas:
-      "SBox-1";
-  }
-  
-  .SBox-1 { grid-area: SBox-1; }
-  
-
+  }
+  @media #{$screen__l} {
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr;
+  }
+  li {
+    padding-top: 5px;
+  }
+}
 
-.Footer-logo {
-    display: flex;
-    background-color: #fff;
-    padding: 50px;
-    margin: 10px;
-    width: 90%;
-       @media (max-width: 600px) {
-            width: 95%;
-        }
+.center__block-1 {
+  place-self: center;
+  a {
+    color: #fff;
+    padding-top: 5px;
+  }
 }
 
-.Footer-Box-1 {
-    display: flex;
-    margin: 10px;
+.center__block-2 {
+  place-self: center;
+  a {
+    color: #fff;
+    padding: 5px;
+  }
 }
-.Footer-Box-2 {
-    display: flex;
-    padding-top: 50px;
-    font-size: 10px;
-    li {
-        padding-top: 10px;
-    }
-    a {
-        color:#fff;
-    }
-    //@media #{$screen__l} 
-    //{
-    //    font-size: 15px;
-    //    padding: 90px 0px 0px 0px;
-    //}
+
+.sb-grid-container {
+  display: grid;
+  width: 100%;
+  height: 100%;
+  grid-template-areas: "side__block-1";
 }
 
-.Footer-Box-3 {
-    display: flex;
-    padding-top: 40px;
-    font-size: 10px;
-    a {
-        color:#fff;
-        border: solid 1px #fff;
-        padding: 10px;
-    }
-    //@media #{$screen__l} 
-    //{
-    //    font-size: 15px;
-    //    padding: 100px 50px 5px 50px;
-    //}
+.side__block-1 {
+  grid-area: side__block-1;
 }
 
+.footer__logo {
+  display: flex;
+  background-color: #fff;
+  padding: 50px;
+  margin: 10px;
+  width: 90%;
+  @media (max-width: 600px) {
+    width: 95%;
+  }
+  @media #{$screen__l} {
+    margin: 20px;
+  }
+}
 
-.FBox-1 {
-    display: flex;
-    margin: 10px;
-    width: 50%;
-    li {
-        margin: 20px;
-    }
-    a {
-        margin: 10px;
-    }
-}
-.FBox-2 {
-    display: flex;
+.footer__block-1 {
```

Comment:
footer-block 1 og 2 gjør også samme jobb. Vurder å bruke en felles footer__block klasse for å legge til flex, margin, og width. Hvis det er noe forskjeller beholder du 
footer__block-1 og 2 klassene i html.

<div class="footer__block footer__block-1"></div>
.footer__block {
 felles CSS
 &-1 {
  spesifikt for footer__block-1
 }
 &-2 {
  spesifikt for footer__block-2
 }
}